src/e2e/java/teammates/e2e/pageobjects/StudentHomePage.java

```diff
@@ -1,10 +1,16 @@
 package teammates.e2e.pageobjects;
 
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
 /**
  * Page Object Model for student home page.
  */
 public class StudentHomePage extends AppPage {
 
+    @FindBy(linkText = "View team")
+    private WebElement studentViewTeamBtn;
```

Comment:
Will this works if there are multiple "View team" buttons?

Reply:
Yes. It’ll populate the first “View team” shown. In this case, it’ll always access the first course shown in student home

Reply:
Normally I'd say this is too use-case specific, e.g. you can't to access the second/third/... course. But since this is E2E test I can close one eye.